plugin/pkg/seed/validator/admission.go

```diff
@@ -135,18 +135,9 @@ func (v *ValidateSeed) Validate(ctx context.Context, a admission.Attributes, o a
 		return apierrors.NewInternalError(err)
 	}
 
-	backupbuckets, err := v.backupBucketLister.List(labels.Everything())
-	if err != nil {
-		return apierrors.NewInternalError(err)
-	}
-
 	if admissionutils.IsSeedUsedByShoot(seedName, shoots) {
 		return admission.NewForbidden(a, fmt.Errorf("cannot delete seed '%s' which is still used by shoot(s)", seedName))
 	}
 
-	if admissionutils.IsSeedUsedByBackupBucket(seedName, backupbuckets) {
```

Comment:
You can completely remove this method - e.g. delete https://github.com/gardener/gardener/blob/e0a0b192fbdc1a3c4329f4a702d4694d9c3d5437/plugin/pkg/utils/miscellaneous.go#L62-L70.